gcc/config/aarch64/cygming.h

```diff
@@ -318,6 +318,15 @@ do {							\
 
 #define HAVE_64BIT_POINTERS 1
 
+extern void i386_pe_asm_lto_start (void);
+extern void i386_pe_asm_lto_end (void);
+
+/* Kludge because of missing PE-COFF support for early LTO debug.  */
+#undef  TARGET_ASM_LTO_START
```

Comment:
Those declarations can also be shared with i386 in `mingw` config.

Reply:
For now we are still quite different from i386/cygming.h and I am trying to keep them alike as much as possible.
Maybe in the feature it will be also reused and it will be easier to make it.

Reply:
I am rather thinging about having both: `mingw/cygming.h` (or even different name) for shared declarations and `i386/cygming.h` and `aarch64/cygming.h` for unique declarations.

Reply:
mingw should be architecture independent. Historically it was created from i386, however it is changing in direction not to have any i386_ prefix or architecture related code there.

Reply:
> mingw should be architecture independent.

This statement is true though I don't understand how it relates to my concern of starting to use shared header for architecture independent code and using separate headers for architecture-dependend code?

Reply:
It is going in that direction, just for now it is too early to that work for cygmin.h.
Also it has lower priority than other work.

Reply:
OK, I am fine with that kind of assurement that the requested change will happen before the patch series 3 submission. So far it was not said explicitly. Thank you.